content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -215,6 +215,50 @@
                 "rest",
                 "v1",
                 "attributes"
```

Comment:
There are missing query parameters on this endpoint GET /api/rest/v1/attributes/{code}
- with_table_select_options

Reply:
Aren't you referencing to https://github.com/akeneo/pim-api-docs/pull/599/files#diff-965e2072337a3d3d1d8e0ba37d6d642f21cbaf9d9c009c75a6fa016b25516cc3R257 ?

Reply:
Yes it is this option but it is also available for the GET of an attribute